ingress/ingress-rules-goquorum.yml

```diff
@@ -28,11 +28,11 @@ spec:
   rules:
   - http:
       paths:
-      - path: /rpcnode
+      - path: /rpc
```

Comment:
`curl -v -X POST -H "Content-Type: application/json" --data '{"jsonrpc":"2.0","method":"eth_blockNumber","params":[],"id":1}' http://<INGRESS_IP>/rpc`

This change is to bring it in line with following command and rpc url